Client/Application/AppLaunchUtil.swift

```diff
@@ -174,4 +180,60 @@ class AppLaunchUtil {
                             category: .sync)
         }
     }
+
+    // MARK: - SKAdNetwork Postbacks
+
+   private struct ConversionValue {
```

Comment:
I think the 2 adNetwork functions can be added to this struct and the struct can be moved to it's own file in order to keep the AppLaunchUtil lean.

Reply:
I moved the functions to the dedicated struct, which has now its own file. Thank you for suggestions!